drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua

```diff
@@ -0,0 +1,86 @@
+-- Copyright 2023 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local clusters  = require "st.zigbee.zcl.clusters"
+local device_management = require "st.zigbee.device_management"
+local log = require "log"
+local Level = clusters.Level
+local OnOff = clusters.OnOff
+local ColorControl = clusters.ColorControl
+
+local emit_pushed_event = function(button_name, device)
+  local additional_fields = {
+    state_change = true
+  }
+  local event = capabilities.button.button.pushed(additional_fields)
+  local comp = device.profile.components[button_name]
+  if comp ~= nil then
+    device:emit_component_event(comp, event) 
+  else
+    log.warn("Attempted to emit button event for unknown button: " .. button_name)
+  end
+end
+
+
+local emit_held_event = function(button_name, device)
+  local additional_fields = {
+    state_change = true
+  }
+  local event = capabilities.button.button.held(additional_fields)
+  local comp = device.profile.components[button_name]
+  if comp ~= nil then
+    device:emit_component_event(comp, event) 
+  else
+    log.warn("Attempted to emit button event for unknown button: " .. button_name)
+  end
+end
+
+local do_configure = function(self, device)
+  device:send(device_management.build_bind_request(device, OnOff.ID, self.environment_info.hub_zigbee_eui))
+  -- The device reports button presses to this group but it can't be read from the binding table
+end
+
+local SLED_button = {
+  NAME = "SLED Button",
+  capability_handlers = {
+    [capabilities.refresh.ID] = {
+      [capabilities.refresh.commands.refresh.NAME] = do_refresh,
```

Comment:
you are also referencing `do_refresh` here

```
Checking drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua 3 warnings

    drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua:30:45: (W612) line contains trailing whitespace
    drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua:44:45: (W612) line contains trailing whitespace
    drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua:59:54: (W113) accessing undefined variable do_refresh
```

Please address these comments. Please write some tests for this behavior.

Reply:
The code was modified and there were no errors through lua check.